tools/bgo_tools/get_projects/test-data/test_credentials.json

```
@@ -0,0 +1,12 @@
+{
+    "dialect" : "postgresql",
+    "host":"localhost",
```

This should be the default IP address of dockers bridge network to the host machine:

```suggestion
    "host":"172.17.0.1",
```

might work if run the tool in the container again.

In case it is not the default on github, then we could determine it in the CI with `ip addr show docker0 | grep "inet\b" | awk '{print $2}' | cut -d/ -f1` and use `sed` to dynamically set it in the json.